book/src/confidence.md

```diff
@@ -0,0 +1,67 @@
+# Confidence
+
+The confidence metric aims to give clients a measure of the network confirmation levels on a particular block.
+
+# Calculation RPC
+
+Clients can request confidence metrics from a validator for a signature `s` through `get_confidence(s: Signature) -> Confidence` over RPC. The `Confidence` struct is of type `HashMap<u32, f32>`. This returned hashmap represents the confidence metric for the particular block `N` that contains the signature `s` as of the last block `M` that the validator voted on. 
```

Comment:
Can you hard-wrap to 80 characters for easier commenting?

Reply:
sure thing, done!